.github/workflows/ci.yaml

```diff
@@ -0,0 +1,101 @@
+name: Publish to ECR
```

Comment:
This name seems a little misleading if this also does testing.

Reply:
i was actually suppose to name this Test, build and publish to ECR ! will update this